setup.py

```diff
@@ -34,6 +33,7 @@
 else:
     _install_requires = [
         'pylint>=1.0',
+        'pylint-plugin-utils>=0.2.4',
```

Comment:
Not sure if this is a reasonable version requirement, but it appears to be the most recent and has been around for nearly a year.